pcdsdevices/gon.py

```diff
@@ -432,12 +437,12 @@ def k_to_e(self, eta=None, kappa=None, phi=None):
         coordinates : tuple
             Spherical coordinates.
         """
-        if not eta and eta != 0:
+        if eta is None:
```

Comment:
This is one of the sloppy things I copied directly for the old code - thanks for cleaning these up

Reply:
Yep, I think we were scared to break things